modules/c3-assisted-installer-preparing-image-converting.adoc

```diff
@@ -6,8 +6,11 @@
 [id="c3-assisted-installer-preparing-image-converting_{context}"]
 = Converting and uploading the image to {oci-c3-no-rt}
 
-Convert the ISO image to an {oci} image and upload it to your {oci-c3-short} system from your OCI Home Region Object 
-Store.
+Convert the ISO image to an {oci} image and upload it to your {oci-c3-short} system from your {oci-first} Home Region Object Store. You can perform this step on your laptop, rather than on the bastion server or within environments such as Oracle Solution Center.
```

Comment:
This is a bit of a nit but maybe "laptop" is a bit informal and also perhaps even too prescriptive, when it might refer to any computer in general? Not sure what the replacement would be though, maybe computer or something. Either way it's not as big of a deal and I'm sure most people would understand well enough